test/mitmproxy/addons/test_view.py

```diff
@@ -546,11 +549,13 @@ def test_configure():
         with pytest.raises(Exception, match="Invalid interception filter"):
             tctx.configure(v, view_filter="~~")
 
-        tctx.configure(v, console_order="method")
+        tctx.configure(v, view_order="method")
         with pytest.raises(Exception, match="Unknown flow order"):
-            tctx.configure(v, console_order="no")
+            tctx.configure(v, view_order="no")
 
-        tctx.configure(v, console_order_reversed=True)
+        tctx.configure(v, view_order_reversed=True)
 
+        console_addon = consoleaddons.ConsoleAddon(tctx.master)
+        tctx.configure(console_addon)
         tctx.configure(v, console_focus_follow=True)
```

Comment:
I think `view_focus_follow` is used by the eventlog addon and the view addon. Maybe just keep it as `console_focus_follow`, but let's have 
```python
        loader.add_option(
            "view_order_reversed", bool, False,
            "Reverse the sorting order."
        )
```
in both addons? alternatively, we could add a `consoleaddons.common_options(loader)` function which we invoke from both `load` calls to express the dependency. In any case, that should solve the issue that we need to add the ConsoleAddon here and it's clear what the addon depends on. What do you think?